tests/mol-embed_fxl/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+        <meta charset="utf-8" />
+		<title>Media overlay with embedded video in FXL</title>
```

Comment:
Let's avoid acronyms in titles — "fixed layout"?

Reply:
+1